chart/elastalert2/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Automated rule-based alerting for Elasticsearch
 name: elastalert2
-version: 2.4.0
+version: 2.4.1
```

Comment:
Please change this back. The versions are adjusted when the admins release the next version.